build/pom.xml

```diff
@@ -48,7 +48,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-dependency-plugin</artifactId>
-                <version>3.6.1</version>
+                <version>3.8.1</version>
```

Comment:
Non-blocking: How does the integration test use Maven such that this change is wanted or needed if we move to `npx` and `oscal-js`?